images/runtime/Dockerfile

```diff
@@ -16,33 +15,43 @@ FROM ${CILIUM_LLVM_IMAGE} as llvm-dist
 FROM ${CILIUM_BPFTOOL_IMAGE} as bpftool-dist
 FROM ${CILIUM_IPROUTE2_IMAGE} as iproute2-dist
 
-FROM --platform=linux/amd64 ${COMPILERS_IMAGE} as cni-builder
+FROM --platform=linux/amd64 ${GOLANG_IMAGE} as gops-cni-builder
```

Comment:
What happened to ${COMPILERS_IMAGE}? We still want to ship a stripped-down compiler right?

Reply:
Yes but not in this PR.